locale/en/docs/guides/simple-profiling.md

```diff
@@ -63,9 +63,10 @@ app.get('/auth', (req, res) => {
     return res.sendStatus(400);
   }
 
-  const hash = crypto.pbkdf2Sync(password, users[username].salt, 10000, 512);
+  const { salt, hash } = users[username];
+  const encryptHash = crypto.pbkdf2Sync(password, salt, 10000, 512, 'sha512');
 
-  if (users[username].hash.toString() === hash.toString()) {
+  if (hash.toString() === encryptHash.toString()) {
```

Comment:
Also unrelated: if `hash` and `encryptHash` are buffers, the best way to check for equality is `hash.equals(encryptHash)`.

Reply:
This should probably be a constant time comparison though? As not to leak the expected hash? (`crypto.timingSafeEqual`)

Reply:
Oh, right, good point.

Reply:
@LinusU: Take your advice :)